shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -876,14 +876,13 @@ - (instancetype)initWithBridge:(fml::WeakPtr<flutter::AccessibilityBridgeIos>)br
                            uid:(int32_t)uid
                   platformView:(nonnull FlutterTouchInterceptingView*)platformView {
   if (self = [super initWithBridge:bridge uid:uid]) {
-    _platformView = [platformView retain];
+    _platformView = platformView;
     [platformView setFlutterAccessibilityContainer:self];
   }
   return self;
 }
 
 - (void)dealloc {
-  [_platformView release];
   _platformView = nil;
```

Comment:
nit: it's not necessary to nil out an ivar here (can remove the whole dealloc).

Reply:
I'm about to remove this dealloc in https://github.com/flutter/engine/pull/52535, I wouldn't worry about it in this PR.

Reply:
Oh actually that's not true, the PR doesn't include `SemanticsObject`.  But I'll get to that one hopefully soon 🙂